로그인&회원가입.js

```diff
@@ -0,0 +1,105 @@
+document.addEventListener("DOMContentLoaded", function () {
+  const emailInput = document.getElementById("email");
+  const passwordInput = document.getElementById("password");
+  const verifypasswordInput = document.getElementById("verifypassword");
+  const loginButton = document.getElementById("login_box");
+
+  const emailError = document.getElementById("error_email");
+  const passwordError = document.getElementById("error_password");
+  const verifypasswordError = document.getElementById("error_verifypassword");
+
+  // 이메일검증 이벤트
+  if (emailInput) {
+    emailInput.addEventListener("focusout", function () {
+      const emilStatus = emailInput.value.trim();
```

Comment:
P2:
emailStatus 라는 이름이 email input의 value를 나타내는지 잘 모르겠습니다.
아래와 같은 이름을 추천드립니다.

```suggestion
      const emailValue = emailInput.value.trim();
```